### PR TITLE
fixing comparison bug

### PR DESCRIPTION
**Pull request by loganwright**

### CODE REVIEW

File: SwiftWisdomTests/VersionTests.swift [hunk 1]
```diff
@@ -35,4 +35,16 @@ class VersionTests: XCTestCase {
         
         XCTAssert(highest > middle)
     }
+    
+    func testVersionLessThan() {
+        let left = Version(string: "1.0.0")!
+        let right = Version(string: "0.5.2")!
+        XCTAssertFalse(left < right)
+    }
```
Comment (Benuuu) [inline SwiftWisdomTests/VersionTests.swift:51]: Could you write some more tests for this that improve the coverage?

Comment (loganwright) [inline SwiftWisdomTests/VersionTests.swift:51]: @Benuuu Sure, any suggestions on what I should add?

Comment (paulrolfe) [inline SwiftWisdomTests/VersionTests.swift:51]: Maybe something with no patch? One with a revision? One with gibberish in the initializer string? `33,64/f4`

**Comment by paulrolfe** [comment]
:shipit:

File: SwiftWisdomTests/VersionTests.swift (hunk 1 above)
Comment (loganwright) [inline SwiftWisdomTests/VersionTests.swift:51]: Boom, done!

**Pull request merged**